app/views/ama_layout/_success.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="row">
+  <div class="large-12">
+    <div class="success_notification"><span><%= flash[:success].html_safe %></span></div>
```

Comment:
The `flash[:success]` would get replaced with `message` (or whatever variable name you used in the "locals" assignment).